final_prep/README.md

```diff
@@ -44,33 +43,32 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+<img width="1440" alt="Screen Shot 2022-03-05 at 12 09 12 PM" src="https://user-images.githubusercontent.com/98728690/156896919-54d20002-d800-4c90-abda-03e5a6de1af0.png">
+
+<img width="1440" alt="Screen Shot 2022-03-05 at 12 09 17 PM" src="https://user-images.githubusercontent.com/98728690/156896943-4edb819f-2cc8-475f-8d54-f8fa3f9f0b82.png">
+
+<img width="1440" alt="Screen Shot 2022-03-05 at 12 09 20 PM" src="https://user-images.githubusercontent.com/98728690/156896953-a9deeb30-5924-4e49-a83b-ab91fc89b04c.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
-
+  - [ ] Add link to your gist here: [My Mentorship Gist] https://gist.github.com/WyattKribs/8fa633a93f45bfae955c59948d7e234b)
```

Comment:
This is GREAT and I love so much that you shared what you are really wanting to grow in, as early as that intro message!!